sdk/core/core-rest-pipeline/src/retryStrategies/throttlingRetryStrategy.ts

```diff
@@ -2,54 +2,77 @@
 // Licensed under the MIT license.
 
 import { PipelineResponse } from "..";
+import { parseHeaderValueAsNumber } from "../util/helpers";
 import { RetryStrategy } from "./retryStrategy";
 
 /**
- * Returns the number of milliseconds to wait based on a Retry-After header value.
- * Returns undefined if there is no valid value.
- * @param headerValue - An HTTP Retry-After header value.
+ * The headers that come back from Azure services representing
+ * the amount of time (minimum) to wait to retry (in milliseconds).
  */
-function parseRetryAfterHeader(headerValue: string): number | undefined {
+const RetryAfterMillisecondsHeaders: string[] = ["retry-after-ms", "x-ms-retry-after-ms"];
```

Comment:
Can avoid needing to do the concat later:
```suggestion
const RetryAfterMillisecondsHeaders: string[] = ["retry-after-ms", "x-ms-retry-after-ms", RetryAfterHeader];
```

Reply:
Then I'll call this array `RetryAfterHeaders` instead of `RetryAfterMillisecondsHeaders`.